### ISSUE TITLE
Smoother navigation bar UX to reduce flickering and missed clicks

### ISSUE THREAD

**Issue by SableRaf** [processing/processing-website#316]
In its current state, the dropdown menu on the navigation bar is prone to flickering, and it is sometimes hard to click the button you want to hit. If your cursor hovers just one pixel outside of the hitbox the menu disappears.

There are a few ways to solve this, from introducing a delay (with the issue that it creates lag) to smarter approaches that analyze mouse movement. Some more details and code examples can be found in this article: https://www.smashingmagazine.com/2021/05/frustrating-design-patterns-mega-dropdown-hover-menus/

**Comment by Dev-Code24** [comment]
is this issue fixed? I cannot see any PR is made referring to this issue.

**Comment by SableRaf** [comment]
Hi @Dev-Code24, this hasn't been fixed yet. Note: this should probably be addressed together with issue https://github.com/processing/processing-website/issues/430 "Improve top bar menu navigation for keyboard and screen reader users"

**Comment by Dev-Code24** [comment]
@SableRaf 
ok, I will fix both issues in a single PR.

**Comment by SableRaf** [comment]
Thanks @Dev-Code24. I assigned this to you. It is not strictly required at the moment, but I think it helps keeping things tidy so I'm trying to get into the habit of assigning issues.

**Comment by SableRaf** [comment]
@Dev-Code24 no problem thanks for looking into it :) If you gathered any insights on the issue feel free to share them.

**Comment by Rahulhanje** [comment]
@SableRaf is this issue fixed ?

**Comment by SableRaf** [comment]
@Rahulhanje Not yet! Let me know if you'd like to take this on.

**Comment by Rahulhanje** [comment]
Yes, sure Since I am a beginner I would be happy if you could guide me on this issue need some time

**Comment by SableRaf** [comment]
@Rahulhanje This issue is a bit more advanced and I'm not sure we have someone available to mentor you through it. I would suggest you start with the issue we already assigned to you and come back to this after the other is resolved or once you have gained a bit more confidence with the codebase. How does that sound?

**Comment by Rahulhanje** [comment]
Ya you guessed it correctly i don't have anyone one to mentor me  so first i will try to fix the first issue you are assigned to me then i will come to this issue 
I am ok with you

**Comment by Rahulhanje** [comment]
@SableRaf 
 "I've completed my first issue, and the PR has been merged. I'd like to give this new issue a try. Could you please assign it to me and provide some ideas on how to solve the error
